docs/framework/wcf/feature-details/how-to-create-a-workflow-service-with-messaging-activities.md

```diff
@@ -16,45 +16,45 @@ This topic describes how to create a simple workflow service using messaging act
   
 3.  When the project is created, the Service1.xamlx file is opened in the designer as shown in the following illustration.  
   
-     ![The default workflow displayed in the designer](../../../../docs/framework/wcf/feature-details/media/defaultworkflowservice.JPG "DefaultWorkflowService")  
+     ![Screenshot that shows the default workflow in the designer.](./media/how-to-create-a-workflow-service-with-messaging-activities/default-workflow-service.jpg)  
```

Comment:
I think we should specify what the screenshot is about in these alt texts

Reply:
@mairaw I've updated this one and the next (Messaging tab > Messaging section) based on your feedback. I'm not sure if I should make additional changes to the alt text content in the rest of this article. It seems that's what you're asking me to do. I'm just not sure what to change them to. Please provide guidance and I'll make the changes.